packages/trader/src/Stores/Modules/Trading/trade-store.ts

```diff
@@ -212,6 +212,7 @@ export default class TradeStore extends BaseStore {
     basis_list: Array<TTextValueStrings> = [];
     currency = '';
     stake_boundary: Partial<TStakeBoundary> = {};
+    trade_stop_out = '';
```

Comment:
@kate-deriv please remove it, it's unused, and please also remove it from packages/trader/src/Modules/Trading/Containers/Multiplier/multiplier-amount-modal.jsx